misc updates (#732)
livechat/livechat-public-docs: content/messaging/agent-chat-api/rtm-reference/index.mdx

<Section>
<Text>

> ### List Customer

Reviewer: 😳

| Object | Notes                        |
| ------ | ---------------------------- |

> | `thread_id`      | present if the chat is active                        |

Reviewer: ```suggestion
| `thread_id`      | Present if the chat is active                        |
```

| `transferred_to` **\*** | IDs of groups and Agents the chat is now assigned to |

> | `queue`          | present if the chat is queued after the transfer     |

Reviewer: ```suggestion
| `queue`          | Present if the chat is queued after the transfer     |
```

**\*)**

> One of the following combinations of ids is possible:

Reviewer: ```suggestion
One of the following combinations of IDs is possible:
```

> - both `group_ids` and `agent_ids`, when the chat was transferred to a group, and an agent was assigned immediately (even if the chat was transferred within the same group),

Reviewer: ```suggestion
- Both `group_ids` and `agent_ids`. In situations when a chat was transferred to a group and it was immediately assigned to an Agent (even if the chat transfer occurred in the same group).
```

> - only `group_ids`, when the chat was transferred to a group and no agents were available, queue will be present,

Reviewer: ```suggestion
- Only `group_ids`. When a chat was transferred to a group and no Agents were available; the queue will be present.
```

Reviewer: Line 3855: 
Only `agent_ids`. When a chat was transferred directly to an Agent; there'll be no queue.
